src/content/docs/tutorials/javareporter/junit.md

```diff
@@ -0,0 +1,156 @@
+type: article
+url: https://docs.testomat.io/tutorials/javareporter/junit
+head:
+  - tag: meta
+    attrs:
+      name: keywords
+      content: JUnit integration, Testomat.io, Java-Reporter, JUnit tests, test reporting, Maven dependency, Gradle dependency, test management, Java test automation, test run customization, test properties configuration
+---
+<!--
+    ## JUnit Integration Setup
+        - Maven/Gradle dependencies
+        - Java-Reporter library setup
+        - Version management
+
+    ## Test Import and Synchronization
+        - Importing JUnit tests to Testomat.io
+        - Using Java-Check-Tests CLI
+        - Cross-platform commands (UNIX/MAC/Windows)
+
+    ## Java-Reporter Configuration
+        - junit-platform.properties setup
+        - Required properties (listening, API key)
+        - Property configuration methods
+        - Environment variables vs JVM properties
+
+    ## Test Run Customization
+        - Basic run properties (title, environment, group)
+        - Advanced properties (shared runs, export options)
+        - Public vs private results
+        - Integration with existing runs
+-->
+
+
+# JUnit integration with Testomat.io
+
+This guide demonstrates how to integrate JUnit with [Testomat.io](https://app.testomat.io) for efficient test management
+and detailed reporting.
+
+---
+
+## Adding the dependency
+
+Add the proper **Maven/Gradle** snippet to your **pom.xml/build.gradle** file.
+
+```xml
+
+<dependency>
+    <groupId>io.testomat</groupId>
+    <artifactId>java-reporter-junit</artifactId>
+    <version>0.7.4</version>
+</dependency>
+```
+
+```properties
+    implementation group: 'io.testomat', name: 'java-reporter-junit', version: '0.7.4'
+```
+
+> NOTE: There might be an updated version of the library published. You can check it in
+> the [Maven Central Repository](https://central.sonatype.com/artifact/io.testomat/java-reporter-junit)
+
+---
+
+## Importing JUnit tests to [Testomat.io](https://app.testomat.io)
+
+It is important to import your test codebase to [Testomat.io](https://app.testomat.io) to see the correct project
+structure and test source code in the UI.
+
+You can easily do this with the following commands:
+
+Here is the command for UNIX/MAC users:
+
+```bash
+  export TESTOMATIO_URL=... && \
+  export TESTOMATIO=... && \
+  curl -L -O https://github.com/testomatio/java-check-tests/releases/latest/download/testomatio.jar && \
+  java -jar testomatio.jar sync
+```
+
+And here is the command for Windows users:
+
+```cmd
+    set TESTOMATIO_URL=...&& ^
+    set TESTOMATIO=...&& ^
+    curl -L -O https://github.com/testomatio/java-check-tests/releases/latest/download/testomatio.jar&& ^
```

Comment:
```
curl -L -O -z testomatio.jar https://github.com/testomatio/java-check-tests/releases/latest/download/testomatio.jar && \
```

The -z flag makes curl only download if the remote file is newer than the local file, or if the local file doesn't exist.